doc/site/content/docs/guides/getting-started/widgets-and-gadgets.md

```diff
@@ -3,21 +3,37 @@ title = "Widgets and Gadgets"
+### Key Areas
+- LuaRules - Generally home to lower level customisations that affect unit behavior or overall game operation
+- LuaGaia - The controller of world objects not owned by a player (e.g wrecks, map features)
+- LuaIntro - The handler for showing the game load screens.
+- LuaMenu - The handler for showing the main menu, both present on game launch (if no script is specified) and can be switched back to at any time during the game.
+- LuaUI - The handler for the in-game UI.
 
-### Synced and Unsynced
+### Entrypoint into Lua code
+Each of these handlers will provide a Lua environment with some predefined globals and start by executing a main.lua file within their respective folders in the game. (LuaGaia and LuaRules have two entry points main.lua for Synced and draw.lua for Unsynced) [more information on the environments and globals provided is available here.](https://springrts.com/wiki/Lua:Environments)
 
-This is an incredibly importand concept in regards to addons. When you write an addon, you are really writing *two* addons: one that runs in a synced context, and one in an unsynced context. But, what does this mean?
-- Synced code is anything that affects the game simulation: ordering units around, etc. since it runs on a fixed tick rate.
-- Unsynced code is anything that doesn't care about the tick rate of the simulation state, like reading unit position, etc.
+To allow you to control and extend the game/engine behavior predefined functions in your lua code will be called (if they are present) by the engine at certain hook points or on events in the engine. These are commonly referred to as call-ins (but could also be known as event handlers, callbacks). The list of available call-ins is extensive and can be retrieved in Lua using Script.GetCallInList()
 
-### Basic Outline
+## Widgets and Gadgets
+Widgets and Gadgets are concepts that have been adopted by several games using Recoil as a modular way to extend the game and are not an engine level concept. Practically at a general functionality level both widgets and gadgets are the same and often abstracted to just the term addon. The different terms are mostly used to logically separate the types of addons.
 
-Both widgets and gadgets follow this basic blueprint:
+Gadgets typically being lower level game logic, unit behaviour functionality that defines your game and should be present for all players, they are typically only found in LuaRules and LuaGaia and are often included using VFS.ZIP_ONLY so as not to be easily overridden by end users (your packaged version takes priority). 
```

Comment:
Some mention that gadgets refer specifically to components under the LuaRules environments, that are loaded and run in both `Synced` and `Unsynced` luarules environments, the fork being to check `gadgetHandler:IsSyncedCode()`. Similar mechanism for luahandler but no game or example addon use luahandler for luarules that I'm aware of, not worth mentioning imo.

Widgets refer specifically to LuaUI components.